### PR TITLE
Added Reading Challenge Info

### PR DESCRIPTION
**Pull request by LeahOlson**

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -115,6 +118,54 @@ <h5 class="text-light h5 font-weight-bold my-4">Teaching girls about technology
...
+          </div>
+        </div>
+
+        <div class="col-10 mx-auto"><hr class="hr"></div>
+
+        <h3 class="text-center mb-4">Sponsors</h3>
+        <div class="col-10 d-flex flex-wrap justify-content-center mx-auto align-items-center">
+
+          <div>
+            <a href="https://www.ameritas.com/" id="Ameritas" target="_blank" rel="noopener">
```
Comment (troynguyen8) [inline src/index.html:161]: Should we also add ameritas to the sponsors section?

File: src/index.html [hunk 2]
```diff
@@ -115,6 +118,54 @@ <h5 class="text-light h5 font-weight-bold my-4">Teaching girls about technology
...
+          <i>
+            April 1, 2021 - May 15, 2021
+          </i>
+        </div>
+
+        <div class="col-12 d-flex justify-content-center">
+          <div class="col-lg-8 col-xs-10">
+            <h3 class="text-center mb-4">Join Girls Code Lincoln and Lincoln City Libraries for a STEM Reading Challenge!</h3>
+            <p >
+              By taking part in this challenge, children can learn about the rich history and
```
Comment (troynguyen8) [inline src/index.html:136]: This text is a tad small - I'd recommend a little bigger!

File: src/index.html [hunk 3]
```diff
@@ -115,6 +118,54 @@ <h5 class="text-light h5 font-weight-bold my-4">Teaching girls about technology
...
+              current contributions of women in Science, Technology, Engineering and Math (S.T.E.M.).
+              This challenge is open to children of all ages in Lincoln.
+            </p>
+            <p>
+              To get started, sign up on the <a href="https://lincolnlibraries.beanstack.org/" target="_blank">Beanstack site</a> or app.
+              To complete the challenge, read four books from our <a href="https://lincolnlibraries.beanstack.org/lists/20851" target="_blank">booklist</a>
+              or choose any book about women in STEM and write reviews.
+            </p>
+            <p>
+              Stop by a library location to pick up an Empowerment bookmark! Everyone who registers and completes the challenge
```
Comment (troynguyen8) [inline src/index.html:146]: ooo should we add some pictures to this section such as a picture of the empowerment bookmark?

**Review by troynguyen8 — APPROVED** [review]
Looks great! Let me know if you have any questions on any comments I made / how to address test failures

File: src/index.html (hunk 3 above)
Comment (LeahOlson) [reply src/index.html:146]: that's a good idea! maybe I'll look into adding that in the future

File: src/index.html (hunk 2 above)
Comment (LeahOlson) [reply src/index.html:136]: oh yeah good point - I'll fix that 👍

**Review by justinjstark — APPROVED** [review]
This all looks great. Well done, Leah.

**Pull request merged**